edu.stanford.Almond.json

```diff
@@ -0,0 +1,272 @@
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "command" : "/app/bin/edu.stanford.Almond",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
+        "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.login1",
+        "--talk-name=org.gtk.vfs.Daemon",
+        "--talk-name=org.gtk.vfs.mountpoint_http",
+        "--talk-name=org.gnome.Shell.Screenshot",
+        "--talk-name=org.freedesktop.secrets"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/lib/cmake",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "/bin/npm",
+        "/bin/cvc4",
+        "/bin/antlr3",
+        "/bin/pi",
+        "/lib/yarn",
+        "/bin/yarn",
+        "/bin/mimic*",
+        "/bin/lfsc-checker",
+        "/bin/compile_regexes",
+        "/bin/t2p",
+        "*.la",
+        "*.a",
+        "Makefile",
+        "binding.gyp",
+        "*.o",
+        "*.d"
+    ],
+    "modules" : [
+        {
+            "name" : "lapack",
+            "buildsystem" : "cmake",
+            "builddir" : true,
+            "config-opts" : [
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DCBLAS=ON",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "build-options" : {
+                "env" : {
+                    "FC" : "/usr/lib/sdk/gfortran-62/bin/gfortran"
+                }
+            },
+            "build-commands" : [
+                "/usr/lib/sdk/gfortran-62/install.sh"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "http://www.netlib.org/lapack/lapack-3.8.0.tar.gz",
+                    "sha256" : "deb22cc4a6120bff72621155a9917f485f96ef8319ac074a7afbc68aab88bcf6"
+                }
+            ]
+        },
+        {
+            "name" : "cln",
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://www.ginac.de/CLN/cln-1.3.4.tar.bz2",
+                    "sha256" : "2d99d7c433fb60db1e28299298a98354339bdc120d31bb9a862cafc5210ab748"
+                }
+            ]
+        },
+        {
+            "name" : "antlr3",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "install -m 0755 -d /app/share/java",
+                "install -m 0644 antlr-3.4-complete.jar /app/share/java",
+                "echo 'java -cp /app/share/java/antlr-3.4-complete.jar org.antlr.Tool \"\\$@\"' > /app/bin/antlr3",
```

Comment:
I think you need Java extension for this to work.

Reply:
Actually, I realized, because cvc4 (which depends on antlr3) is built from the tarball, it does not call the parser generator, only link to the runtime library. So I can probably remove this.